src/components/stats/PlayersGameStats.js

```diff
@@ -2,6 +2,7 @@ import { useState } from "react";
 import { useSelector } from "react-redux";
 import classes from "./PlayersGameStats.module.css";
 import PlayersGameStatsList from "./PlayersGameStatsList";
+import { separatePlayersToTeams } from "../../services/teamGameStatsService";
```

Comment:
it's not really a service - it's kind of helper. I would put it in helpers sub-directory under the component directory, I think.